<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIFI_Conf</class>
 <widget class="QDialog" name="WIFI_Conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{Background-color:rgb(238, 238, 238)};</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>460</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_Settings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>44</y>
      <width>310</width>
      <height>420</height>
     </rect>
    </property>
    <property name="title">
     <string>Wifi Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>34</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>75</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subnet mask:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>113</y>
       <width>114</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gateway Address:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIPaddress">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>29</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSubnet">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>70</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtGateway">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>111</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>208</y>
       <width>110</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Wireless - Ad-hoc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>165</y>
       <width>110</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>Wireless - (SSID)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpbxSecurity">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>244</y>
       <width>300</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Security</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>39</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbEncryption">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>31</y>
        <width>160</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtKey">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wireless - Key</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="txtSSID">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>157</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbAdhoc">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>201</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnOk">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>369</y>
       <width>89</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string> Ok</string>
     </property>
     <property name="shortcut">
      <string>F2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnCancel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>369</y>
       <width>89</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string> Cancel</string>
     </property>
     <property name="shortcut">
      <string>F4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ON_OFF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>310</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_WIFI">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>15</y>
       <width>46</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WIFI </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnON_OFF">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>8</y>
       <width>93</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BtnON_OFF</tabstop>
  <tabstop>txtIPaddress</tabstop>
  <tabstop>txtSubnet</tabstop>
  <tabstop>txtGateway</tabstop>
  <tabstop>txtSSID</tabstop>
  <tabstop>cmbAdhoc</tabstop>
  <tabstop>cmbEncryption</tabstop>
  <tabstop>txtKey</tabstop>
  <tabstop>BtnOk</tabstop>
  <tabstop>BtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
